### PR TITLE
add parameter q

### LINKED ISSUE #8: Parameter -q

### PR DESCRIPTION
**Pull request by justinmueller05082000**
fixes #8

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ func main() {
 	wordsParameter := flag.Bool("w", false, "use PATTERN that only matches words.")
 	linesParameter := flag.Bool("x", false, "use PATTERN that only matches whole lines.")
 	ignoreCaseParameter := flag.Bool("i", false, "ignore case distinctions")
+	quietParameter := flag.Bool("i", false, "suppress all normal output")
```
Comment (Al2Klimov) [inline main.go]: Srsly?!

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (justinmueller05082000) [reply main.go]: im tired

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [x] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD master)..HEAD^)`
* [x] `git commit --amend --no-edit`
* [x] `git push -f origin feature/add-parameter-q-8`

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Write a test protocol.

**Comment by justinmueller05082000** [comment]
**Test protocol:**

**toGrep.txt:**
```
a
a
b
a
a
a
b
b
```

**Terminal:** `go run . -q -e 'a' < toGrep.txt`
**Output:**
```
(empty)
```

**Terminal:** `grep -q -e 'a' < toGrep.txt`
**Output:**
```
(empty)
```

**Result:**
Both outputs are equal.

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**